zenodo/modules/profiles/templates/zenodo_profiles/orcid_profile.html

```diff
@@ -0,0 +1,221 @@
+{#-
+# This file is part of Zenodo.
+# Copyright (C) 2017 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+
+{% extends "zenodo_theme/page.html" %}
+
+{%- block page_body %}
+<div class="container">
+  <div class="row" id="orcid-profile">
+    <div class="col-sm-3 col-xs-12" id="profile" style="margin-bottom:20px">
+      <br />
+      <img height="400px" width="400px"
```

Comment:
Some padding around the image would be useful, since now it takes way too much space compared to the rest of the page and is not as important in our use-case compared to eg. a Facebook or Twitter profile.